
	public void sample()throws Exception {
		try {
			int num =55;
			
			if(num > 30) {
				throw new Exception();
			}
		}catch(Exception e) {
			System.out.println("例外発生！！");
	}
			
	}
}